Description: avoids xdrs being used without being initialised first. Without this
patch dsniff segfaults when decoding RPC packets on x86_64.
Author: Matthew Boyle <mlb@decisionsoft.co.uk>
Origin: http://pkgs.fedoraproject.org/cgit/rpms/dsniff.git/tree/dsniff-2.4-rpc_segfault.patch
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/rpc.c
+++ b/rpc.c
@@ -125,6 +125,9 @@
 			return (0);
 		}
 	}
+	else
+		return (0);
+	
 	stat = xdr_getpos(&xdrs);
 	xdr_destroy(&xdrs);
 	
